src/classes/CRLP_RollupProcessor.cls

```diff
@@ -499,14 +516,37 @@ public inherited sharing class CRLP_RollupProcessor {
         SObject updatedRecord = handler.getPopulatedSObject();
 
         // Determine if the updated SObject record is different than the parent.
-        // If not, set it to null
-        if (CRLP_Rollup_SVC.resultsNeedUpdate(parent, updatedRecord, rollupDefs) == false) {
+        Boolean needsUpdate = CRLP_Rollup_SVC.resultsNeedUpdate(parent, updatedRecord, rollupDefs);
+
+        // Unconditionally set force skew flag if parent is Account/Contact and running in skew mode
+        if (mode == CRLP_RollupProcessingOptions.BatchJobMode.SkewMode && this.useForceSkewModeField && !this.isSetSkewModeField) {
+            updatedRecord.put(this.qualifiedSkewField, true);
+            needsUpdate = true;
+        }
+
+        if (!needsUpdate) {
             updatedRecord = null;
         }
 
         return updatedRecord;
     }
 
+    /**
+     * @description Set variables associated with handling force skew mode checkbox on Account and Contact
+     * @param parent Base SObject for the dynamically generated query
+     */
+    private void getSkewModeValues(SObject parent) {
+        if (this.qualifiedSkewField == null) {
+            this.isSetSkewModeField = false;
+            this.qualifiedSkewField = UTIL_Namespace.StrTokenNSPrefix('CustomizableRollups_UseSkewMode__c');
+            this.useForceSkewModeField = (parent.getSObjectType() == Account.SObjectType || parent.getSObjectType() == Contact.SObjectType);
+            if (this.useForceSkewModeField && parent.isSet(this.qualifiedSkewField))
+            {
+                this.isSetSkewModeField = (Boolean) parent.get(this.qualifiedSkewField);
+            }
```

Comment:
This particular block shouldn't be in this method because it will vary for each individual parent record. It should be up in the `completeRollupForSingleSummaryRecord` method.

Reply:
Good catch.  Thanks.  That also means your earlier suggestion to update skew values once should ignore isSetSkewModeField and leave that to be handled on a record-by-record basis.